hash/msx2_flop.xml

```diff
@@ -15724,6 +16025,59 @@ The following floppies came with the machines.
 		</part>
 	</software>
 
+	<software name="yumepro">
+		<description>Yume Pro RPG Shaon-ban (Japan)</description>
+		<year>1994</year>
+		<publisher>Syntax</publisher>
+		<info name="alt_title" value="夢プロRPG謝恩版"/>
+		<info name="usage" value="Requires a Japanese system."/>
+		<part name="flop1" interface="floppy_3_5">
+			<dataarea name="flop" size="737280">
+				<rom name="yume pro rpg shaon-ban - syntax.dsk" size="737280" crc="b5582441" sha1="cea1b78601854b2541d5c9741dc6dd68f6778000"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="yumeji">
+		<description>Yumeji Asakusa-Kitan (Japan)</description>
+		<year>1992</year>
+		<publisher>Fairytale</publisher>
+		<notes>Does not work on all systems.</notes>
+		<info name="alt_title" value="夢二・浅草綺譚"/>
+		<info name="barcode" value="4988715001986"/>
+		<info name="usage" value="Requires a Japanese system."/>
+		<part name="flop1" interface="floppy_3_5">
+			<feature name="part_id" value="Disk A"/>
+			<dataarea name="flop" size="1020496">
+				<rom name="yumeji - fairytale (disk 1).dmk" size="1020496" crc="2990b188" sha1="7d53a1d112e24948e65e8f1eac28af130d116220"/>
+			</dataarea>
+		</part>
+		<part name="flop2" interface="floppy_3_5">
+			<feature name="part_id" value="Disk B"/>
+			<dataarea name="flop" size="737280">
+				<rom name="yumeji - fairytale (disk 2).dsk" size="737280" crc="034ae3a7" sha1="ba780be3eca3b6109d50f5eea4232609dc18d23d"/>
+			</dataarea>
+		</part>
+		<part name="flop3" interface="floppy_3_5">
+			<feature name="part_id" value="Disk C"/>
+			<dataarea name="flop" size="737280">
+				<rom name="yumeji - fairytale (disk 3).dsk" size="737280" crc="e9b7b349" sha1="88ca21a4e75303ff10e9aa963d800195a4416535"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="yuureikun">
+		<description>Yuurei-Kun (Japan)</description>
+		<year>1989</year>
+		<publisher>System Sacom</publisher>
+		<info name="alt_title" value="幽霊君"/>
```

Comment:
Should be “Yuurei-kun” (or “Yūrei-kun”).

Reply:
Updated the description